src/main/java/org/opentripplanner/model/TripAlterationOnDate.java

```diff
@@ -0,0 +1,56 @@
+package org.opentripplanner.model;
+
+import org.opentripplanner.model.calendar.ServiceDate;
+
+import java.util.Objects;
+
+
+/**
+ * This class represent a trip scheduled on a given date. The trip alteration is cancellation,
+ * planned, extraJourney, or replaced.
+ * <p>
+ * This entity is ONLY used if the input date is NeTEx and the NeTEx data have
+ * DatedServiceJourneys. There is a one-to-one relationship between instances of this class
+ * and instances of the NeTEx DatedServiceJourney.
+ */
+public class TripAlterationOnDate extends IdentityBean<AgencyAndId> {
+  private final AgencyAndId id;
+  private final TripAlteration alteration;
+  private final ServiceDate date;
+  private final TripAlterationOnDate replaces;
```

Comment:
Can a TripAlteration replace several TropAlterations?

Reply:
I see now that alterations are replaced based on Id, but that this is separate from a Trip replacing another Trip.